lib/player.js

```diff
@@ -1450,6 +1450,15 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         }
       }
 
+      // On PlayStation, cached MediaKeySystemAccess objects may corrupt
+      // after several playbacks, and they are not able anymore to properly
+      // create MediaKeys objects. To prevent it, clear the cache after
+      // each playback.
+      if (shaka.util.Platform.isPS4() || shaka.util.Platform.isPS5()) {
```

Comment:
Since this can happen on STBs, could you add it as a config? Thanks!

Reply:
Done!